sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -706,7 +728,9 @@ export class StreamingReceiver extends MessageReceiver {
 
     let sReceiver: StreamingReceiver;
 
-    if (options?._createStreamingReceiver) {
+    if (context.streamingReceiver) {
```

Comment:
Should this check also include a check in `context.streamingReceiver.isOpen()`?
Take a case where there was a non retryable error on the receiver link or that there was a retryable error and all our retries are exhausted. The user's onError handler would get called. Then they make another call to `subscribe()`. At that point, here, would `context.streamingReceiver` still point to the old broken link?

Reply:
Check my logic on this - .init() will get called on this (in the retry code below) so it will do the normal logic of checking to see if the receiver is open (if it's not then it'll go and do the normal init logic and create a receiver). 

If it _is_ open then we just fall through.

I added in a pair of tests in streamingReceiver.spec.ts for it:
* `create() with an existing receiver and that receiver is NOT open()create() with an existing receiver and that receiver is NOT open()`
* `create() with an existing receiver and that receiver is open()create() with an existing receiver and that receiver is open()`

Reply:
ok, makes sense